src/ansi-c/builtin_factory.cpp

```diff
@@ -0,0 +1,159 @@
+/*******************************************************************\
+
+Module:
+
+Author: Daniel Kroening, kroening@kroening.com
+
+\*******************************************************************/
+
+#include "builtin_factory.h"
+#include "ansi_c_internal_additions.h"
+
+#include "ansi_c_parser.h"
+#include "ansi_c_typecheck.h"
+
+#include <cstring>
+#include <ostream>
+#include <sstream>
+
+const char windows_headers[]=
+  "int __noop();\n"
+  "int __assume(int);\n";
```

Comment:
Why is this here and not in `ansi_c_internal_additions.*`?

Reply:
moved